src/components/views/messages/MImageBody.js

```diff
@@ -302,11 +303,14 @@ export default class MImageBody extends React.Component {
                 onMouseLeave={this.onImageLeave} />;
         }
 
+        if (this._isGif() && !SettingsStore.getValue("autoplayGifsAndVideos") && !this.state.hover) {
+            gifLabel = <p className="mx_MImageBody_gifLabel">GIF</p>;
```

Comment:
Tiny nit, ``<p>`` vs ``<span>``?

Reply:
Apologies for the delay. I personally don't see any problem with `<p>`, it seems to work fine and I don't see it causing any problems in future.